src/client/transaction.go

```diff
@@ -0,0 +1,298 @@
+// cluster and return the expected response.
+func (c APIClient) AppendCreateBranch(txn *transaction.Transaction, request *pfs.CreateBranchRequest) (*types.Empty, error) {
+	return c.appendTransactionHelper(
+		txn,
+		&transaction.TransactionRequest{
+			Request: &transaction.TransactionRequest_CreateBranch{
+				CreateBranch: request,
+			},
+		},
+	)
+}
+
+// AppendDeleteBranch is a wrapper function for the AppendTransaction RPC which
+// will append a DeleteBranchRequest to an existing transaction in the Pachyderm
+// cluster and return the expected response.
+func (c APIClient) AppendDeleteBranch(txn *transaction.Transaction, request *pfs.DeleteBranchRequest) (*types.Empty, error) {
+	return c.appendTransactionHelper(
+		txn,
+		&transaction.TransactionRequest{
+			Request: &transaction.TransactionRequest_DeleteBranch{
+				DeleteBranch: request,
+			},
+		},
+	)
+}
+
+// AppendCopyFile is a wrapper function for the AppendTransaction RPC which
+// will append a CopyFileRequest to an existing transaction in the Pachyderm
+// cluster and return the expected response.
+func (c APIClient) AppendCopyFile(txn *transaction.Transaction, request *pfs.CopyFileRequest) (*types.Empty, error) {
+	return c.appendTransactionHelper(
+		txn,
+		&transaction.TransactionRequest{
+			Request: &transaction.TransactionRequest_CopyFile{
+				CopyFile: request,
+			},
+		},
+	)
+}
+
+// AppendDeleteFile is a wrapper function for the AppendTransaction RPC which
+// will append a DeleteFileRequest to an existing transaction in the Pachyderm
+// cluster and return the expected response.
+func (c APIClient) AppendDeleteFile(txn *transaction.Transaction, request *pfs.DeleteFileRequest) (*types.Empty, error) {
+	return c.appendTransactionHelper(
+		txn,
+		&transaction.TransactionRequest{
+			Request: &transaction.TransactionRequest_DeleteFile{
+				DeleteFile: request,
+			},
+		},
+	)
+}
+
+// AppendStartCommit is a wrapper function for the AppendTransaction RPC which
+// will append a StartCommitRequest to an existing transaction in the Pachyderm
+// cluster and return the expected response. This is a rare write operation that
+// actually has a return value - the ID of the new Commit.
+func (c APIClient) AppendStartCommit(txn *transaction.Transaction, request *pfs.StartCommitRequest) (*pfs.Commit, error) {
+	info, err := c.TransactionAPIClient.AppendTransaction(
+		c.Ctx(),
+		&transaction.AppendTransactionRequest{
+			Transaction: txn,
+			Items: []*transaction.TransactionRequest{
+				&transaction.TransactionRequest{
+					Request: &transaction.TransactionRequest_StartCommit{
+						StartCommit: request,
+					},
+				},
+			},
+		},
+	)
+	if err != nil {
+		return nil, grpcutil.ScrubGRPC(err)
+	}
+
+	switch response := info.Responses[len(info.Responses)-1].Response.(type) {
+	case *transaction.TransactionResponse_Commit:
+		return response.Commit, nil
+	}
+	return nil, fmt.Errorf("added to transaction but received an unexpected response type")
+}
+*/
```

Comment:
These should probably be removed or uncommented, it'll bitrot if it's left like this.

Reply:
:+1: - this was just old code I left in for reference while refactoring - deleted.